mirrord/agent/src/steal/ip_tables.rs

```diff
@@ -220,24 +226,38 @@ where
     }
 }
 
+#[derive(Debug)]
 pub(crate) enum IPTableFormatter {
     Normal,
-    Mesh,
+    Mesh(String),
 }
 
 impl IPTableFormatter {
     const MESH_OUTPUTS: [&'static str; 2] = ["-j PROXY_INIT_OUTPUT", "-j ISTIO_OUTPUT"];
+    const MESH_NAMES: [&'static str; 2] = ["PROXY_INIT_OUTPUT", "ISTIO_OUTPUT"];
 
     #[tracing::instrument(level = "trace", skip_all)]
     pub(crate) fn detect<IPT: IPTables>(ipt: &IPT) -> Result<Self> {
         let output = ipt.list_rules("OUTPUT")?;
 
-        if output.iter().any(|rule| {
+        if let Some(mesh_ipt_chain) = output.iter().find_map(|rule| {
```

Comment:
Probably good to add a comment here mentioning that we need this for the health probes.